core/src/main/java/org/owasp/dependencycheck/data/nodeaudit/NodeAuditSearch.java

```diff
@@ -127,6 +142,14 @@ public NodeAuditSearch(Settings settings) throws MalformedURLException {
                         final NpmAuditParser parser = new NpmAuditParser();
                         return parser.parse(jsonResponse);
                     }
+                case 503:
+                    LOGGER.debug("Invalid payload submitted to Node Audit API. Received response code: {} {}",
+                            conn.getResponseCode(), conn.getResponseMessage());
+                    if (count > 5){
+                        return submitPackage(packageJson, ++count);
+                    }
+                    throw new SearchException("Could not perform Node Audit analysis. Invalid payload submitted to Node Audit API.");
```

Comment:
Again - I don't think we should be calling this an invalid payload?

```suggestion
                    throw new SearchException("Could not perform Node Audit analysis - service returned a 503.");
```

Reply:
Yeah sorry copy and paste mistake ..